.github/workflows/release.yml

```diff
@@ -17,41 +18,43 @@ env:
 
 jobs:
   check-version-changed:
-    if: github.event_name != 'pull_request' || contains(github.event.pull_request.labels.*.name, 'release-test')
     runs-on: ubuntu-24.04
     permissions:
       contents: write
+    env:
+      CURRENT_TAG: ${{ github.ref_name }}
     outputs:
-      TAG: ${{ steps.versions.outputs.TAG }}
-      PKG_VER: ${{ steps.versions.outputs.PKG_VER }}
       RELEASE_NOTES: ${{ steps.versions.outputs.RELEASE_NOTES }}
     steps:
       - uses: actions/checkout@v4
-        with:
-          fetch-tags: true
-          fetch-depth: 0
 
-      - name: Check Versions
+      - name: Check that tag and version in Cargo.toml match
```

Comment:
Nit: Shouldn't this go into the description?

Reply:
What do you mean? Neither jobs nor steps don't have a description. I can leave a comment before the step.

Reply:
What I mean is that this reads like a description now while it is supposed to be a title

Reply:
A comment would also be fine